toml.md

```diff
@@ -161,9 +161,24 @@ In JSON land, that would give you the following structure:
 }
 ```
 
-Whitespace around dot-separated parts is ignored, however, best practice is to
+Whitespace around dot-separated parts is ignored. However, best practice is to
 not use any extraneous whitespace.
 
+```toml
+fruit.name = "banana"     # this is best practice
+fruit. color = "yellow"    # same as fruit.color
+fruit . flavor = "banana"   # same as fruit.flavor
+```
+
+Indentation is treated as whitespace. Tabs and spaces before keys are ignored.
+You shouldn't use them. But if you must, use them sparingly.
```

Comment:
I would delete this line (and similar insertions elsewhere in the PR). Whether and how to indent is a stylistic issue and we as spec writers should not take a line here. OUR style in the spec should be not to indent – I agree on that. But if others want to use indentation and consider is good for readability, who are we to prevent/discourage them?

Reply:
That line was a holdover from a previous draft. I just forgot to clean it up. I'll do that now.